css/style.css

```diff
@@ -1,3 +1,26 @@
-h1 {
-  color: salmon;
+
+
+body {
+  background:#EDEFEC;
+  text-align: center;
+  /*display:flex;
+  flex-direction: column;
+  justify-content: center;*/
+  /*margin:0 auto;*/
+}
+.content {
+  max-width: 700px;
+  margin:0 auto;
+  display:flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.imagebox1, .imagebox2 {
+  display:flex;
+  justify-content:space-around;
+}
+img {
```

Comment:
Watch out with using selectors that are overly broad. In this assignment you only had 6 `img` elements, but in a larger site with several pages that load images we could get into trouble if some of those other pages need images to display at different sizes.

It's rarely a bad idea to be extra specific with something like `div.imagebox div.box img` Specificity ensures we don't style beyond our intention. And you could always peel off some of the layers later if you find them to be extraneous, and maybe boil it down something like `.box img`